src/libDirectoryService/DSBlockPostProcessing.cpp

```diff
@@ -231,8 +231,11 @@ void DirectoryService::SendDSBlockToShardNodes(
       std::vector<Peer> shardDSBlockReceivers;
 
       LOG_GENERAL(
-          INFO, "Sending to {NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD} peers : "
-                    << NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD);
+          INFO,
+          "Sending message with hash: ["
+              << DataConversion::Uint8VecToHexStr(this_msg_hash).substr(0, 6)
+              << "] to NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD:"
+              << NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD << " shard peers");
```

Comment:
I suggest this message to move down to log numOfDSBlockReceivers, because the value of  NUM_FORWARDED_BLOCK_RECEIVERS_PER_SHARD we can know from constants.xml.

Reply:
This was added to log after suggestion from previous review to clearly show which constants we are using on DS side (sender) and Shard side (receiver) so that we know we are not using different constants.